org.nicolasbock.ebuildtester.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: org.nicolasbock.ebuildtester
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: ebuildtester
+finish-args:
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
what is this for?

Reply:
This app uses `docker` to create a container. As far as I understand, `flatpak`'s architecture does not allow the use of host owned files without this addition.